submissions/iva-stasia/js-dom/js/script.js

```diff
@@ -0,0 +1,117 @@
+    description3: 'Spike walked Ein later as bait for Hakim, only for Ein to be drawn to a dog whistle from the research scientists who were still looking for him. Hakim saw Ein run by, and was able to grab him, rendering him unconscious, and drive away in a stolen car. Ein woke up, however, and interfered with Hakim\'s driving as he avoided Spike in the Swordfish II. Ein jumped out of the car and was fortunately saved by Spike. With Hakim captured, Ein was then part of the crew, despite Spike\'s dislike of animals.',
+    image: './img/ein_main.png',
+    subtitle: 'Personality',
+    extra: 'Being a dog, Ein isn\'t exactly loaded with personality hooks. He tends to act like your average everyday dog—most of the time. However, he\'s very smart, and understands people perfectly, and will react to people who react negatively to him. He gets along quite well with Ed, and the two are close friends.'
+  }
+];
+
+const menuLessBttn = document.querySelector('.nav_less_bttn');
+const menu = document.querySelector('.nav_bar');
+
+menuLessBttn.addEventListener('click', () => {
+  menu.classList.toggle('nav_narrow');
+});
+
+const tabs = document.querySelectorAll('.tab');
+const mainContent = document.querySelector('.main');
+
+
+const generateContent = ({name, quote, description1, description2, description3, image, subtitle, extra}) => {
+  return `
+    <article class="article">
+      <h1 class="char_name">${name}</h1>
+      <blockquote class="quote">${quote}</blockquote>
+      <div class="char_description">
+        <p class="descr">${description1}</p>
+        <p class="descr">${description2}</p>
+        <p class="descr">${description3}</p>
+      </div>
+    </article>
+    <img src="${image}" class="char_img" alt="" />
+    <div class="char_extra">
+      <h2 class="exrta_info_title">${subtitle}</h2>
+      <p class="extra_info">
+        ${extra}
+      </p>
+    </div>
+  `;
+}
+
+mainContent.innerHTML = generateContent(characters.find(content => content['id'] === 1));
+
+menu.addEventListener('click', (event) => {
+  const tabActive = event.target.closest('.tab')
+  const tabId = +event.target.closest('.tab').dataset.tabTarget;
```

Comment:
Added <code>if</code> and deconstruction. But I left <code>const tabId = target.closest('.tab').dataset.tabTarget;</code> because if I remove <code>closest</code>, then when I click, for example, on a icon, the value will return as undefined.